freesound/settings.py

```diff
@@ -275,6 +281,10 @@
 MIRROR_ANALYSIS = None  # list of locations to mirror contents of SOUNDS_PATH, set to None to turn off
 MIRROR_AVATARS = None  # list of locations to mirror contents of AVATARS_PATH, set to None to turn off
 
+# Footer banner
+FOOTER_BANNER_CONTENTS = \
+    '<a href="javascript:void(0);" onclick="openSurveyPage();hideFooterBanner();setSurveyVisited();">' \
+    '<img src="/media/images/content/freesound-survey-2017.png"/></a>'  # Set to None for no banner
```

Comment:
sure, makes sense!
I can do this change too for the survey code, which I hope to release soon (this week)